### PR TITLE
Linux Docker Compose

### PR DESCRIPTION
**Pull request by gambol99**
- adding a linux compatible version of the docker-compose

### CODE REVIEW

**Comment by gambol99** [comment]
for @danielwhatmuff :-)

**Comment by danielwhatmuff** [comment]
Linking to #29

**Comment by danielwhatmuff** [comment]
> Is it possible to share parts of the YML, to avoid duplication?

I guess we could generate it based on the main one using script/task/template or something but do you think it's worth it..? I think linking to the open issues leading to the extra compose DNS stuff might be a good shout though..?

**Comment by lewismarshall** [comment]
Very close overlap work - #213

Regarding:
> Is it possible to share parts of the YML, to avoid duplication?

`docker-compose -f` merges multiple files - see:

https://docs.docker.com/compose/reference/overview/#specifying-multiple-compose-files

>When you supply multiple files, Compose combines them into a single configuration. Compose builds the configuration in the order you supply the files. Subsequent files override and add to their predecessors. The docker-compose.yml file might specify a webapp service.

**Comment by danielwhatmuff** [comment]
Removed duplication of config to ease maintaining versions etc.

File: README.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ The hub should then be available at http://localhost:3000
 ### Dependent services
 
 A database, mock user service and auth proxy can all be run locally using Docker Compose, using the provided `docker-compose.yml`.
+Note for linux users you can use `docker-compose -f docker-compose.yml -f docker-compose-linux.yml <commands>`. This is required due to the current lack of support for `host.docker.internal` in [Docker for Linux](https://github.com/docker/for-linux/issues/264).
```
Comment (jits) [inline README.md]: Could you add an extra linebreak here so Markdown doesn't combine this line with the one above.

Comment (danielwhatmuff) [reply README.md]: Done

**Pull request closed without merging**